test/localhub/external_storage_backend_test.go

```diff
@@ -0,0 +1,371 @@
+package localhub
```

Comment:
This test was for showcase purposes and doesn't contain proper assertions. Please remove `fmt.Printf/Println` and add proper assertions for us to make sure it works as expected 🙂